recipes/sshpass/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "sshpass" %}
+{% set version = "1.06" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://downloads.sourceforge.net/project/{{ name }}/{{ name }}/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c6324fcee608b99a58f9870157dfa754837f8c48be3df0f5e2f3accf145dee60
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkgconfig
+
+test:
+  commands:
+    - test -x ${PREFIX}/bin/sshpass
+
+about:
+  home: https://sourceforge.net/projects/sshpass/
+  license: GPL-2.0+
```

We prefer SPDX identifiers in the hope of one day having consistent metadata everywhere :)
```suggestion
  license: GPL-2.0-or-later
```